test/fixtures/dialects/postgres/postgres_values_alias.sql

```diff
@@ -0,0 +1,4 @@
+select *
+from (
+    values (1, 2), (3, 4)
+) as t(c1, c2);
```

Comment:
is `as t (c1, c2)` valid (i.e. with a space)?

If not is there anyway for SQLFluff to prevent parsing that? Cause at the moment I think it will accept it when it maybe shouldn't?

Reply:
@tunetheweb yeah no spaces is fine 😄 
![image](https://user-images.githubusercontent.com/80432516/150551229-a78a0aa7-bbc9-4a29-bd48-1db49908dc0c.png)

For general interest though, yes you can prevent SQLFluff from parsing gaps.
You would use the `allow_gaps=False` paramater to the grammar segment.
e.g. Hypothetically if this grammar didn't allow spaces (it does but let's imagine):
```python
match_grammar = Sequence(
        Ref.keyword("AS", optional=True),
        Sequence(
            Ref("SingleIdentifierGrammar"),
            Bracketed(Ref("SingleIdentifierListSegment"), optional=True),
            allow_gaps=False,
        ),
    )
```
😎

Reply:
Excellent. Approval coming up!